pandas/tests/arrays/interval/test_interval.py

```diff
@@ -443,3 +443,19 @@ def test_arrow_interval_type_error_and_warning():
     msg = "Argument `closed` is deprecated in favor of `inclusive`"
     with tm.assert_produces_warning(FutureWarning, match=msg, check_stacklevel=False):
         ArrowIntervalType(pa.int64(), closed="both")
+
+
+@pytest.mark.parametrize("timezone", ["UTC", "US/Pacific", "GMT"])
+@pytest.mark.parametrize("inclusive", ["both", "neither", "left", "right"])
```

Comment:
Can you use the `closed` fixture instead?

Reply:
i think we just renamed that one right? (to inclusive) agreed should use the fixture

Reply:
Yeah or the `inclusive_endpoints_fixture`

Reply:
Ok Have updated